packages/frontend-2/lib/acc/graphql/fragments.ts

```diff
@@ -0,0 +1,23 @@
+import { graphql } from '~~/lib/common/generated/gql'
+
+export const projectAccSyncItemFragment = graphql(`
```

Comment:
unless you need the fragment/query/mutation/gql doc definitions available from multiple locations (e.g. a shared query), you should keep it as a private (non-exported) implementation detail of the component/composable that owns it

goes for all fragments, queries and mutations. i'd prefer it if we'd move these now, cause then its gonna be easier to delete them after (cause they're colocated with the rest of the code)